### DISCUSSION TITLE
How to get the baseline light curve?

### DISCUSSION THREAD

**Discussion by hmjin1** [phoebe-project/phoebe2#594] [General]
Hi, I'm a Phoebe user, i have a question. i use Phoebe get a binary's light curve, it's flux default consider the ellipsoidal,reflection,boosting effect ...,reference ‘The theory and design principles of PHOEBE’   Figure 5.8.  I want to get a baseline light curve ,how to set the parameter and set which parameter when get  the light curve?  Any wisdom would be greatly appreciated!
![baseline](https://user-images.githubusercontent.com/48662425/168991390-fde70d84-bda1-4a7d-97db-7a5a1f4bd551.png)

**Comment by kecnry** [comment]
Baseline here just means the same system parameters but without distortion (to remove ellipsoidal contributions) and without reflection & boosting.  As of the 2.2 release PHOEBE [currently no longer supports boosting](http://phoebe-project.org/docs/latest/tutorials/beaming_boosting), so you don't need to do anything unless you're using an older version (in which case you can set `boosting_method='none'`).  To disable [reflection/irradiation](http://phoebe-project.org/docs/latest/tutorials/reflection_heating), you can set `irrad_method='none'`, and to turn off [surface distortion/ellipsodal effects](http://phoebe-project.org/docs/latest/examples/detached_rotstar), you can set `distortion_method='rotstar'` or `distortion_method='sphere'`.

Hope this helps!
Kyle

**Reply by hmjin1** [reply]
Thank you very much! Follow your advice, i get the baseline lc.It is much to be regretted that  i could not meet Phoebe earlier,  it's a very useful software in simulation.
